### ISSUE TITLE
Sitemap does not include child routes in bag

### ISSUE THREAD

**Issue by kdubious** [OrchardCMS/OrchardCore#7805]
The sitemap tool does not seem to pull the items from a Bag, even though the Autoroute settings are configured for the parent and children.

**Comment by deanmarcussen** [comment]
That is correct. The BagPart would need a feature to handle managing the contained items into a sitemap.
